### PR TITLE
cleaned up tons of garbages and splitted the gigantic api

### PR DESCRIPTION
**Pull request by Vane11ope**

### CODE REVIEW

**Comment by XVilka** [comment]
Thank you! While you are working on the Visual Panels, can you please check Unicode bug with them? https://github.com/radare/radare2/issues/2032 (only V! + unicode). As you have seen it is temporarily disabled for Visual Panels. Would be awesome to have it working.

**Comment by Vane11ope** [comment]
cool i made it my next task. let me fix it up after this getting merged!

File: libr/core/panels.c [hunk 1]
```diff
@@ -493,46 +503,48 @@ static void panel_continue(RCore *core) {
...
+	layout = 0;
 	callgraph = 0;
 	_core = core;
-
-	OS_INIT ();
-	w = r_cons_get_size (&h);
+	menu_pos = 0;
+	menu_x = 0;
+	menu_y = 0;
+	callgraph = 0;
```
Comment (XVilka) [inline libr/core/panels.c]: Duplication - see just a few lines up.

Comment (Vane11ope) [reply libr/core/panels.c]: .........right i'm sorry i can't believe i missed it. i'd delete that no sooner than i get home...... thank you:)

**Comment by Vane11ope** [comment]
done

**Pull request merged**